### PR TITLE
Core Concepts week 1 day 1 updated

### PR DESCRIPTION
**Pull request by beth-panx**
---

⚠️ Merge/Release date 2/14

---

Updates:
- Week 1 day 1 article
- Titles of first week in articles and kickoff
- Banner images

### CODE REVIEW

File: docs/30DaysOfPWA/core-concepts/01.md [hunk 1]
```diff
@@ -1,15 +1,132 @@
...
+ * On newer devices - it can detect device form factors and *deliver responsive experiences* that align with native behaviors.
+ * On modern browsers - it can detect support for Service Workers and Web App Manifest, and use those to unlock and support features like _installability_ and _offline operation_ - just like native apps.
+
+As platforms evolve, and more platforms move to the right (in capabilities support), PWA development will become key to delivering scalable experiences across the broadest range of devices.
+
+---
+
+## How does PWA compare to web and native experiences?
+
+Progressive Web Apps have the ability to get the best of both worlds - broad reach and deep capability - assuming that platforms will continue to evolve to support these open web technologies. Here are a few benefits:
```
Comment (zateutsch) [inline docs/30DaysOfPWA/core-concepts/01.md:57]: The bit about "assuming support" threw me off a little here. Felt like it introduces a level of uncertainty around PWAs that might put a damper on interest.

On the other hand, it is a valid point that may add credibility.

Could see it going either way

File: docs/30DaysOfPWA/core-concepts/01.md [hunk 2]
```diff
@@ -1,15 +1,132 @@
...
+
+In the next post we'll **deconstruct Progressive Web Apps (PWA)** to understand the core technologies that power our implementation, along with a peek at key developer tools to help our understanding.
+
+---
+
+## Exercise: Install your first PWA
+
+Ready to dive into your first exercise? Try this: 
+ 1. Choose a [sample PWAs](https://aka.ms/learn-PWA/30Days-1.1/docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/demo-pwas) and explore the live demo.
+ 2. Try [installing it](https://aka.ms/learn-PWA/30Days-1.1/docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/ux#installing-a-pwa)  on desktop and on mobile. How does the experience differ? How is it similar?
```
Comment (zateutsch) [inline docs/30DaysOfPWA/core-concepts/01.md:127]: Maybe some hints here could guide the exercise?

You could repeat from the list you had earlier about "trying offline" or maybe have them look at for OS capabilities.

**Review by zateutsch — APPROVED** [review]
I think the first article looks great!

I left a couple comments, but they are mostly just my own thoughts while reading, I think everything would do as is 😄 

Didn't see anything grammar/editing wise.

File: docs/30DaysOfPWA/core-concepts/01.md [hunk 3]
```diff
@@ -1,15 +1,132 @@
...
+
+This means that users on modern browsers and newer devices will get an _enhanced_ experience that rivals the native (installed) apps on that platform -- but users on older browsers or devices will continue to get a _usable_ experience in the form of familiar websites.
+
+This allows Progressive Web Apps to combine the **reach** (broad availability and access) of website experiences with the **capability** (hardware features and rich resources) of platform-native experiences. 
+
+**How does it do that?** Let's talk about progressive enhancement!
+
+## What is Progressive Enhancement?
+ 
+[Progressive enhancement](https://aka.ms/learn-PWA/30Days-1.1/alistapart.com/article/understandingprogressiveenhancement) is a design philosophy that puts emphasis on _content-first_ experiences where developers
```
Comment (aarongustafson) [inline docs/30DaysOfPWA/core-concepts/01.md:33]: 🥰  You could also link https://developer.mozilla.org/en-US/docs/Glossary/Progressive_Enhancement

File: docs/30DaysOfPWA/core-concepts/01.md (hunk 1 above)
Comment (aarongustafson) [reply docs/30DaysOfPWA/core-concepts/01.md:57]: Maybe a slight rephrase?

> by offering a solid baseline experience that improves based on each browser’s and OS’s growing support for these open web technologies.

File: docs/30DaysOfPWA/core-concepts/01.md [hunk 4]
```diff
@@ -1,15 +1,132 @@
...
+---
+
+## How does PWA compare to web and native experiences?
+
+Progressive Web Apps have the ability to get the best of both worlds - broad reach and deep capability - assuming that platforms will continue to evolve to support these open web technologies. Here are a few benefits:
+
+ * _They have the reach of websites_. They can be indexed by search engines, discovered or shared by URL, and used from any device with a browser.
+ * _They can behave like native apps_. They can work offline, be installed on device, handle push notifications, and access rich device hardware.
+ * _They can use rich platform capabilities_. Ex: on Windows, PWA can be added to Start Menu, pinned to Taskbar, provide Share Targets for other apps & be published in Microsoft Store with other Windows apps.
+ * _They can reduce costs_. PWAs are developed for all platforms from one codebase - not only does this reduce maintainability costs, it requires a single developer team vs. multiple platform-specific teams, to deliver the same experiences.
```
Comment (aarongustafson) [inline docs/30DaysOfPWA/core-concepts/01.md:62]:
```suggestion
 * _They can reduce costs_. PWAs are developed for all platforms from one codebase - not only does this reduce maintainability costs, it requires a single development team vs. multiple platform-specific teams, to deliver the same experiences.
```

File: docs/30DaysOfPWA/core-concepts/01.md [hunk 5]
```diff
@@ -1,15 +1,132 @@
...
+![DevTools Tips PWA](_media/devtooltips.png)
+
+What makes this a good PWA experience? Because PWA is based on a design philosophy, there are no prescriptive rules - only [desirable characteristics](https://aka.ms/learn-PWA/30Days-1.1/docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/#characteristics-of-a-progressive-web-app-pwa) that make experiences comparable favorably to existing native and web apps:
+
+| Characteristic | Why Is This Desirable? |
+|:---|:---|
+| Discoverability | I can find PWAs in app stores, or via web search. |
+| Installability | I can add PWAs to my device home screen, and launch them - like any native app. |
+| Re-Engageability | I can get push notification alerts (even if PWA is not actively in use) - like any native app . |
+| Network Independence | I can get a usable PWA experience, even in flaky or offline network conditions - like any native app. |
```
Comment (aarongustafson) [inline docs/30DaysOfPWA/core-concepts/01.md:79]: I’ve had some native apps perform really poorly under flakey network conditions, so I don’t know that this is a given :-)

**Review by aarongustafson — APPROVED** [review]
Some minor suggestions.

File: docs/30DaysOfPWA/core-concepts/01.md (hunk 1 above)
Comment (beth-panx) [reply docs/30DaysOfPWA/core-concepts/01.md:57]: @aarongustafson I like this. Should this be added after the "assuming..." sentence? Can you make a suggestion edit?

Comment (beth-panx) [inline docs/30DaysOfPWA/core-concepts/01.md:57]:
```suggestion
Progressive Web Apps have the ability to get the best of both worlds by offering a solid baseline experience that improves based on each browser’s and OS’s growing support for these open web technologies.. Here are a few benefits:
```

File: docs/30DaysOfPWA/core-concepts/01.md [hunk 6]
```diff
@@ -1,15 +1,132 @@
...
```
Comment (beth-panx) [inline docs/30DaysOfPWA/core-concepts/01.md:79]:
```suggestion
| Network Independence | I can get a usable PWA experience - like any native app. |
```

**Pull request merged**